### DISCUSSION TITLE
Coding question

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#33753] [Other Feature Feedback, Questions, & Ideas]
Hey guys,

How many coding languages are present in this world? Name any 10 except HTML, CSS, and JS.

Happy learning...✌🏻

**Comment by radar-blip** [comment]
This site lists a few (but not all):  https://www.tiobe.com/tiobe-index/

And here, if you want a number: https://www.programmingcube.com/how-many-programming-languages-are-in-the-world/

**Reply by samuel-ma** [reply]
Smart @linuxlawson 
Giving him everything but not the actual answer he wants😂

**Reply by cryptofan430** [reply]
https://en.wikipedia.org/wiki/List_of_programming_languages
